spec/std/digest/sha1_spec.cr

```diff
@@ -23,15 +41,19 @@ describe Digest::SHA1 do
         end
       end
 
-      bytes.to_slice.hexstring.should eq(hexdigest)
+      bytes.hexstring.should eq(hexstring)
     end
 
-    it "does hexdigest for #{string.inspect}" do
-      Digest::SHA1.hexdigest(string).should eq(hexdigest)
+    it "does final.hexstring for #{string.inspect}" do
```

Comment:
This spec test `Digest::SHA1.hexdigest` and `final.hexstring` only indirectly. IMO this should either keep the old description or change the implementation to call `final.hexstring` explicitly. I'd prefer the former.